streaming/index.js

```diff
@@ -147,7 +147,7 @@ const redisConfigFromEnv = (env) => {
   }
 
   if (redisNamespace) {
-    redisParams.keyPrefix = redisNamespace;
+    redisParams.keyPrefix = `${redisNamespace}:`;
```

Comment:
Of course as soon as I post that I realise what was missing.

Reply:
This was correct - but only partially. ioredis doesn't automatically prefix `subscribe()` and such.

Reply:
Still not quite there yet.